### PR TITLE
okcomputer app monitoring at /status

### PR DESCRIPTION
**Pull request by ndushay**
connected to #706

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ purl_conn_timeout: 2
 purl_feedback_email: ''
 valid_purl_url: <%= /https?:\/\/purl\.stanford\.edu\/*/ %>
 stacks_url: 'https://stacks.stanford.edu'
-iiif_stacks_url: 'https://stacks.stanford.edu'
+iiif_stacks_url: 'https://stacks.stanford.edu' # unused???
```
Comment (tingulfsen) [inline config/settings.yml]: I agree that it's good to call out unused code, but maybe we can work together to figure out if we can delete this one?

Comment (jkeck) [inline config/settings.yml]: Yeah, it doesn't look like it's used anymore (I'm guessing it did at one point before we swapped out the image viewer).

I say kill it.

Comment (ndushay) [inline config/settings.yml]: done.   This also made me think:  we're starting to get better at documenting why we include gems in a Gemfile.  Should we not be better about documenting un-obvious things in a settings file?   Like "valid_purl_url" and "embed_iframe_url" -- those aren't obvious to me.  (might the latter be better named "iframe_url_prefix" or something?)  If you agree, maybe we should 
* make a ticket for this?
* watch for this on PRs in the future

File: config/initializers/okcomputer.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+require 'okcomputer'
+
+# /status for 'upness' (Rails app is responding), e.g. for load balancer
+# /status/all to show all dependencies
+# /status/<name-of-check> for a specific check (e.g. for nagios warning)
+OkComputer.mount_at = 'status'
+OkComputer.check_in_parallel = true
+OkComputer::Registry.deregister "database" # don't check (unused) ActiveRecord database conn
```
Comment (jkeck) [inline config/initializers/okcomputer.rb]: Are you dead-set against having a consistent style for quotes in this file?

File: config/settings.yml (hunk 1 above)
Comment (jkeck) [inline config/settings.yml]: 👍 I'm not sure we need a ticket officially for this.  But I do like being more careful to document these things in the future (similarly to how we do for the Gemfile).

**Comment by tingulfsen** [comment]
I'm 👍  on this now.

**Pull request merged**